mount/titus-mount-nfs.c

```diff
@@ -305,15 +305,15 @@ int main(int argc, char *argv[])
 
 	/* Now we can switch net/mount namespaces so we can lookup the ip and eventually mount */
 	switch_namespaces(pidfd);
-	fprintf(stderr, "titus-mount: user-inputed options: %s\n", options);
+	fprintf(stderr, "titus-mount-nfs: user-inputed options: %s\n", options);
```

Comment:
Maybe it makes sense to take some time and write a macro here?

Reply:
Yes. I can do that next time.